src/Features/CSharp/Portable/CodeRefactorings/InlineTemporary/InlineTemporaryCodeRefactoringProvider.cs

```diff
@@ -69,7 +69,7 @@ internal partial class InlineTemporaryCodeRefactoringProvider : CodeRefactoringP
                 return;
             }
 
-            if (variableDeclarator.Initializer.Kind() == SyntaxKind.RefExpression)
+            if (variableDeclaration.Type.Kind() == SyntaxKind.RefType)
             {
                 // TODO: inlining byref temps is NYI
```

Comment:
What bug is tracking that this still doesn't work though? @gafter's original concern is that this should work, and that it was being blocked was incorrect. Can this TODO be updated with the tracking bug?